src/transformers/models/tapas/tokenization_tapas.py

```diff
@@ -729,6 +738,19 @@ def batch_encode_plus(
             **kwargs,
         )
 
+    def _get_question_tokens(self, query):
+        """Tokenizes the query, taking into account the max and min question length."""
+
+        query_tokens = self.tokenize(query)
+        if self.max_question_length is not None and len(query_tokens) > self.max_question_length:
+            logger.warning("Skipping query as its tokens are longer than the max question length")
+            return "", []
+        if self.min_question_length is not None and len(query_tokens) < self.min_question_length:
+            logger.warning("Skipping query as its tokens are shorter than the min question length")
```

Comment:
Should it really be a warning when it is doing exactly what the user asks for? I feel like it should be the debug level, not even an info.